src/sparseml/transformers/sparsification/obcq/obcq.py

```diff
@@ -34,7 +34,7 @@
     llama_forward,
     opt_forward,
 )
-from sparseml.transformers.utils.model import SparseCausalLM
+from sparseml.transformers.utils.sparse_model import SparseCausalLM
```

Comment:
Was there a reason not to replace this with the SparseAutoModel? Other than the OPT edge case, I think we only need to add in precision support